README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
```

Comment:
LGTM
Only include one React component per file.
However, multiple Stateless, or Pure, Components are allowed per file. eslint: react/no-multi-comp.

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Readable, Secure, Efficient, Elegant, Reusable, Fault-tolerant, Maintainable